app/views/support_interface/docs/qualifications.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render 'docs_navigation', title: 'Qualifications' %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2" id="gcses">GCSEs</h2>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: GCSE_SUBJECTS, label: 'GCSE subjects') %>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: SINGLE_GCSE_GRADES, label: 'Single GCSE grades') %>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: DOUBLE_GCSE_GRADES, label: 'Double GCSE grades') %>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2" id="a-levels">A levels</h2>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: A_AND_AS_LEVEL_SUBJECTS, label: 'A and AS level subjects') %>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: A_LEVEL_GRADES, label: 'A level grades') %>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: AS_LEVEL_GRADES, label: 'AS level grades') %>
+  </div>
+</div><div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2" id="degrees">Degrees</h2>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: HESA_DEGREE_TYPES.map(&:third), label: 'HESA degree types') %>
+    <%= render SupportInterface::QualificationDocumentationComponent.new(list: HESA_DEGREE_GRADES.map(&:second), label: 'HESA degree grades') %>
```

Comment:
I'm not sure what the use cases for this are, but I'm wondering if it's worth having the associated HESA codes here too?

`
SupportInterface::QualificationDocumentationComponent.new(list: HESA_DEGREE_GRADES.map { |hesa_grade| "#{hesa_grade.second}  - #{hesa_grade.first"), label: 'HESA degree grades') 
`
There's also a definite chance that there's no need for this though.

Reply:
Sure thing. I've just added that in

![image](https://user-images.githubusercontent.com/33458055/112161052-a9a58980-8be2-11eb-94d9-baa39576f75d.png)

Reply:
looks great 👍